src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -63,7 +63,7 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
   const { classes, snapshot } = props;
   // @ts-ignore
   const sourceDataset = snapshot.source[0].dataset;
-  const linkToBq = snapshot.accessInformation?.bigQuery !== undefined;
+  const linkToBq = snapshot.cloudPlatform === 'gcp';
```

Comment:
Is there a situation where the `cloudPlatform` is `gcp` but it is not possible to link to Big Query?

Reply:
not really. this page gets hit if you have access and on GCP you should always have access to the BQ DB at that point